frontend/app/src/components/About.vue

```diff
@@ -35,12 +35,23 @@
                 </td>
                 <td>
                   <div class="d-flex flex-row">
-                    <div class="text-truncate" :class="css.directory">
-                      {{ dataDirectory }}
-                    </div>
+                    <v-tooltip top>
+                      <template #activator="{ on }">
+                        <div
+                          class="text-truncate"
+                          :class="css.directory"
+                          v-on="on"
+                        >
+                          {{ dataDirectory }}
+                        </div>
+                      </template>
+                      <span :class="css.directory">
+                        {{ dataDirectory }}
+                      </span>
+                    </v-tooltip>
                     <v-spacer />
                     <div v-if="isPackaged" class="ml-2">
-                      <v-tooltip top open-delay="400">
+                      <v-tooltip top>
```

Comment:
Is there any reason the open-delay is removed from the tooltip?

Please keep the open delay at 400 ms in tooltips, otherwise, the tooltip will open even if you scroll over the element by accident which can be really annoying.